packages/components/src/Form/Input.tsx

```diff
@@ -70,7 +71,7 @@ const BaseInput: React.FC<Props> = (props) => {
 
 const Input: React.FC<Props> = (props) => {
   if (!props.addonAfter && !props.addonBefore) {
-    return <BaseInput {...props} />;
+    return <BaseInput {...props} className={props.className} />;
```

Comment:
Just question: why you add `className` if it already in props ?

Reply:
Since className prop isn't applied in the `BaseInput` tag, then we need to apply it here :)
I wanted to apply className to the top of the tag.

Reply:
I see, thank you!